@stellar/freighter-api/src/index.ts

```diff
@@ -1,6 +1,7 @@
+import { getAddress } from "./getAddress";
 import { getPublicKey } from "./getPublicKey";
 import { signTransaction } from "./signTransaction";
-import { signBlob } from "./signBlob";
+import { signMessage } from "./signMessage";
```

Comment:
another significant change here is that I'm repurposing `signBlob` to become `signMessage`. We've talked about removing `signBlob` for a bit, and since `signMessage` is a very similar API, I'm making that change now